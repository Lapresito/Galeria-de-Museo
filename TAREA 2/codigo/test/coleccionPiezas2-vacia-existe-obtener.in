crearColeccionPiezas
esVaciaColeccionPiezas
existePiezaColeccionPiezas 50
crearFecha 1/1/2001
crearPieza 10 RaraEspadaDesquiciada Luciana Diaz
agregarColeccionPiezas
esVaciaColeccionPiezas
existePiezaColeccionPiezas 10
existePiezaColeccionPiezas 5
obtenerPiezaColeccionPiezas 10
crearFecha 1/1/2001
crearPieza 6 RaraManianaFunebre Luciana Diaz
agregarColeccionPiezas
esVaciaColeccionPiezas
existePiezaColeccionPiezas 10
existePiezaColeccionPiezas 5
existePiezaColeccionPiezas 6
obtenerPiezaColeccionPiezas 10
obtenerPiezaColeccionPiezas 6
crearFecha 1/1/2001
crearPieza 15 RaraManianaFunebre Luciana Diaz
agregarColeccionPiezas
esVaciaColeccionPiezas
existePiezaColeccionPiezas 10
existePiezaColeccionPiezas 5
existePiezaColeccionPiezas 6
existePiezaColeccionPiezas 15
obtenerPiezaColeccionPiezas 10
obtenerPiezaColeccionPiezas 15
liberarColeccionPiezas
esVaciaColeccionPiezas
existePiezaColeccionPiezas 10
Fin
