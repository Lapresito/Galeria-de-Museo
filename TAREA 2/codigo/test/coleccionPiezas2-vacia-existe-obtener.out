Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Coleccion de piezas creada de forma exitosa.
2> La coleccion de piezas es vacia.
3> La pieza con id 50 NO pertenece a la coleccion.
4> Fecha creada en forma exitosa.
5> Pieza creada en forma exitosa.
6> Pieza agregada de forma exitosa.
7> La coleccion de piezas NO es vacia.
8> La pieza con id 10 pertenece a la coleccion.
9> La pieza con id 5 NO pertenece a la coleccion.
10> Pieza: 10
RaraEspadaDesquiciada-Luciana Diaz
Creada en: 1/1/2001
11> Fecha creada en forma exitosa.
12> Pieza creada en forma exitosa.
13> Pieza agregada de forma exitosa.
14> La coleccion de piezas NO es vacia.
15> La pieza con id 10 pertenece a la coleccion.
16> La pieza con id 5 NO pertenece a la coleccion.
17> La pieza con id 6 pertenece a la coleccion.
18> Pieza: 10
RaraEspadaDesquiciada-Luciana Diaz
Creada en: 1/1/2001
19> Pieza: 6
RaraManianaFunebre-Luciana Diaz
Creada en: 1/1/2001
20> Fecha creada en forma exitosa.
21> Pieza creada en forma exitosa.
22> Pieza agregada de forma exitosa.
23> La coleccion de piezas NO es vacia.
24> La pieza con id 10 pertenece a la coleccion.
25> La pieza con id 5 NO pertenece a la coleccion.
26> La pieza con id 6 pertenece a la coleccion.
27> La pieza con id 15 pertenece a la coleccion.
28> Pieza: 10
RaraEspadaDesquiciada-Luciana Diaz
Creada en: 1/1/2001
29> Pieza: 15
RaraManianaFunebre-Luciana Diaz
Creada en: 1/1/2001
30> Coleccion liberada con exito.
31> La coleccion de piezas es vacia.
32> La pieza con id 10 NO pertenece a la coleccion.
33> Fin.
